backend/templates/includes/navbar.html

```diff
@@ -0,0 +1,18 @@
+<header class="main-header">
+  <h1>PokeBattle</h1>
+</header>
+
+<nav>
+  <ul class="navbar">
+    <li class="navbar-item"><a href="/create-battle">Create a Battle</a></li>
+    <li class="navbar-item"><a href="/settled-battles">Settled Battles</a></li>
+    <li class="navbar-item"><a href="/">Active Battles</a></li>
```

Comment:
For the links, you can use the [Django URL template tag](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#url)
```suggestion
    <li class="navbar-item"><a href="{% url 'create_battle' %}">Create a Battle</a></li>
    <li class="navbar-item"><a href="{% url 'settled_battle' %}">Settled Battles</a></li>
    <li class="navbar-item"><a href="{% url 'home' %}">Active Battles</a></li>
```